### DISCUSSION TITLE
[Benchmark] HP Elitedesk 800 G1 Mini PC

### DISCUSSION THREAD

**Discussion by jedarden** [blakeblackshear/frigate#3743] [General]
I thought I'd put together some discussion on the hardware setup I'm using along with the cameras so others looking to build out their own Frigate-powered NVRs can size their installations accordingly. 

Server Hardware:
- HP Elitedesk 800 G1 Mini PC
- Intel i5-4590T
- 8GB RAM
- 500GB SSD
- 8TB External USD HDD
- USB Coral TPU

Camera brand and detection resolution:
- Amcrest 2592x1944
- Amcrest 2592x1944
- Amcrest 1920x1080
- Hikvision 2688x1520
- Hikvision 2688x1520
- Generic 704x576

Server Usage: 
- Load Average: 3.48 4.76 6.33
- Memory: 2.79/7.7G

Stats:
- Inference Speed 12.66ms

**Comment by mihow** [comment]
Thanks for sharing! Do the Elitedesk Mini's support any of the other Coral TPU form factors besides the USB version? like the M.2?

**Reply by jedarden** [reply]
There's an M.2 slot, but it's for wifi. I don't think it will support M.2 Coral TPU

**Reply by erkexzcx** [reply]
> but it's for wifi

Isn't this M.2 E-key slot which is exactly the one needed for dual TPU accelerator? People are buying [this](https://www.makerfabs.com/dual-edge-tpu-adapter.html) so they can connect it to PCIe, but your situation sounds like you wouldn't need this adapter and can directly connect it to your PC.

If I am not mistaken, wifi/bluetooth is connected on (mostly) laptops via M.2 E-key connector.

**Reply by NickM-27** [reply]
That is incorrect, the adapter is needed because the vast majority of m.2 e keys only support a single bus meanwhile 2 bus are required for 2 corals to be identified.

**Comment by mihow** [comment]
Thanks for your reply. I've read some people saying it works and some it doesn't ha. USB is likely the safer bet.

**Comment by bugsysiegals** [comment]
How many Watts does consume?
